src/torchcodec/decoders/_core/video_decoder_ops.py

```diff
@@ -135,6 +136,7 @@ def add_video_stream_abstract(
     num_threads: Optional[int] = None,
     dimension_order: Optional[str] = None,
     stream_index: Optional[int] = None,
+    device: Optional[str] = None,
```

Comment:
Eventually we'll want to accept a `torch.device` object as well as a str, but I assume that's for later?

Reply:
I think for now we can only accept strings/tensors/ints, not a torch.Device. That's part of the limitation of C++ custom ops AFAIU.